src/packages/@ncigdc/components/analysis/ValidateGeneExpression.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
New component for the third step of the Gene Expression flow, aka validation.
To be fully implemented at a later stage.